src/python/ensembl/production/hive/ensembl_genome_metadata/MetadataUpdaterHiveProcessDb.py

```diff
@@ -0,0 +1,64 @@
+#  See the NOTICE file distributed with this work for additional information
+#  regarding copyright ownership.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+import re
+import json
+from ensembl.production.hive.BaseProdRunnable import BaseProdRunnable
+from eHive import ParamException
+from sqlalchemy.engine.url import make_url
+
+class MetadataUpdaterHiveProcessDb(BaseProdRunnable):
+    def fetch_input(self):
+  try:
+	payload = json.loads(self.param_required("payload"))
+	self.param("comment", payload["input"]["comment"])
+	self.param("database_uri", payload["input"]["database_uri"])
+	self.param_required("database_uri")
+	self.param("email", payload["input"]["email"])
+	self.param("metadata_uri", payload["input"]["metadata_uri"])
+	self.param_required("metadata_uri")
+	self.param("source", payload["input"]["source"])
+	self.param("timestamp", payload["input"]["timestamp"])
+except json.JSONDecodeError as e:
```

Comment:
There is an indentation issue here. Did you set up a test pipeline to make sure it worked?